### PR TITLE
Convert WINDOWS_HOWTO.rtf to Markdown

### PR DESCRIPTION
**Pull request by jrmuizel**
This makes it easier to read on github

### CODE REVIEW

File: WINDOWS_HOWTO.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+1.Compile the windows MSR driver (msr.sys) with Windows DDK Kit (see the sources in the WinMSRDriver directory). For Windows 7 you have to sign the msr.sys driver additionally ([http://msdn.microsoft.com/en-us/library/ms537361(VS.85).aspx](%22http://)).
+
+2.Build the pcm.exe utility in the PCM_Win using Microsoft Visual Studio
+
+3.Copy the msr.sys driver and pcm.exe into a single directory
+
+4.Run pcm.exe utility from this directory
+
+For Windows 7 and Windows Server 2008 R2 the PCM utilities need to be run as administrator:
+
```
Comment (opcm) [inline WINDOWS_HOWTO.md]: could you add the picture from the original RTF to the repository and refer it here?

Comment (jrmuizel) [reply WINDOWS_HOWTO.md]: Done.

Comment (opcm) [reply WINDOWS_HOWTO.md]: > Done.

I don't see any update for your change. Something went wrong?

Comment (opcm) [reply WINDOWS_HOWTO.md]: @jrmuizel I like your change. Is it possible to submit a revised change?

**Comment by jrmuizel** [comment]
I pushed to the wrong branch. It should be there now.

**Pull request merged**